SRVKE-930: Sinkbinding page improvements (#38797)
openshift/openshift-docs: modules/serverless-sinkbinding-reference.adoc

----
{ "extensions": { "extra": "this is an extra attribute", "additional": "42" } }
----

> == The include label

Reviewer: ```suggestion
== Include label
```

I don't think you need "The" here. Looking at the ones above it seems that convention is to specify the item then type of item so in this case, "Include label" or above case "Subject parameter".

> To use a SinkBinding, you need to do assign the `bindings.knative.dev/include: "true"` label to either the resource or the namespace. If the resource definition does not include the label, a cluster administrator can attach it to the namespace by running:

Reviewer: ```suggestion
To use a SinkBinding, assign the `bindings.knative.dev/include: "true"` label to either the resource or the namespace. If the resource definition does not include the label, a cluster administrator can attach it to the namespace by running the following command:
```

Suggestion for minimalistic writing and a complete sentence.
